Tests/SwiftParserTest/translated/PoundAssertTests.swift

```diff
@@ -18,8 +18,12 @@ final class PoundAssertTests: XCTestCase {
   func testPoundAssert1() {
     AssertParse(
       """
-      #assert(true, 123)
-      """
+      #assert(true, 1️⃣123)
+      """,
+      diagnostics: [
+        DiagnosticSpec(message: "expected string literal in '#assert' directive"),
+        DiagnosticSpec(message: "unexpected code '123' in '#assert' directive"),
```

Comment:
Two diagnostics here is a little weird. Maybe we should just eat the next unexpected after a missing token and then have the fix-it remove the unexpected? I haven't checked whether that would make other cases worse though.